src/cond.d

```diff
@@ -25,6 +25,20 @@ import ddmd.root.outbuffer;
 import ddmd.tokens;
 import ddmd.visitor;
 
+private __gshared Identifier idUnitTest;
```

Comment:
The cached objects, named as `idUnitTest` and `idAssert` in here, are also used in `parseVersionCondition()`. I thought seriously about the code duplication, then I concluded that I don't touch them in this PR.
That's the reason I don't remove the use of `Token.toChars`.